### ISSUE TITLE
Use sysfs API for querying watchdog if /dev/watchdog is unavailable

### ISSUE THREAD

**Issue by wiktor-k** [karelzak/util-linux#804]
Hello,

I was using `wdctl` to see watchdog properties but after enabling `RuntimeWatchdogSec` in systemd, systemd takes control of `/dev/watchdog` and `wdctl` prints only:

```
# wdctl
wdctl: /dev/watchdog: watchdog already in use, terminating.
wdctl: cannot open /dev/watchdog: Device or resource busy
```

It was suggested on [systemd-devel mailing list](https://lists.freedesktop.org/archives/systemd-devel/2019-May/042783.html) that wdctl could use sysfs in that case to query watchdog properties:

```
grep . /sys/class/watchdog/watchdog0/*
```

**Comment by karelzak** [comment]
Excellent idea. 

We already have a rich internal API to read from /sys, so it will be trivial to implement it. It's too late for v2.34, but I'll do it for v2.35. Thanks!

**Comment by wiktor-k** [comment]
No problem, in the meantime I'm reading from `/sys` but it would be nice for consistency if `wdctl` worked in all circumstances. I was quite surprised when I enabled `RuntimeWatchdogSec` in systemd and `wdctl` stopped working :)

Have a nice day! :wave:

**Comment by karelzak** [comment]
Merged for v2.35. 

Note that the current /sys does not provide all necessary information. The most missing is info about supported features (aka "options" in ioctl API), but it seems kernel developers are ready to fix this issue.
https://lore.kernel.org/linux-watchdog/20190617135703.l3pzb3ndzhqxhflq@ws.net.home/T/#m2cf39a5951a770c2e3f36189dbe0175b2684362a

For now, the default is still ioctl with fallback to /sys on EBUSY or EACCES.
